R/prepare_data.R

```diff
@@ -0,0 +1,50 @@
+#' Function for reading in and formatting required datasets
+#'
+#' @return 3 datasets of use rate by country, Loess curve of access against nets 
+#' per capita, and estimated median net retention times by country
+#' 
+#' @importFrom utils 'read.csv'
+#' @export
+
+prepare_data <- function() {
```

Comment:
Can we put all variables that the user might want to change, or that might have updated defaults in the future as arguments for this function.
e.g. data location address strings, year filters etc.

Reply:
I broke up the functions into the individual conversion steps and added some tests for each. Documentation should be up-to-date